libsrc/telestrat/orixhdr.s

```diff
@@ -29,7 +29,7 @@
     .byte   $00                     ;  reserved
     .byte   $00                     ;  auto or not
 
-    .word   __BASHEAD_START__       ; Address of start of file
+    .word   __MAIN_START__          ; Address of start of file
```

Comment:
Please settle on comments starting with upper case or not and make the necessary changes to be consistent.